### PR TITLE
2x2 puzzle-grid with background images in correct positions

### PR DESCRIPTION
**Pull request by nat-sharpe**
@mjlohmeier Look ok?

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+var showList = function (image, i) {
```
Comment (xmd404) [inline scripts/main.js]: @nat-sharpe please put scripts.js into js folder :)

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    var thumbImage = $('<img>');
+    var thumbHover = $('<div>');
+    var thumbCaption = $('<p>');
+    var currentI = i;
+
+    thumbnail.addClass("thumbnail");
+    thumbImage.attr('src', image.url);
+    thumbImage.addClass("thumb-image");
+    thumbHover.addClass("thumb-hover");
+    thumbCaption.text(image.caption);
```
Comment (xmd404) [inline scripts/main.js]: @nat-sharpe do we need a caption? curiosity :)

**Review by xmd404 — COMMENTED** [review]
wrong files.

**Comment by nat-sharpe** [comment]
@theWebsiteKid @mjlohmeier review please?

**Review by mjlohmeier — APPROVED** [review]
Needs to be changed to JS in order to be compatible with API

**Pull request merged**